plugins/python/io/NumpyReader.cpp

```diff
@@ -71,6 +75,14 @@ static PluginInfo const s_info
     ""
 };
 
+struct NumpyReader::Args
+{
+    std::string module;
+    std::string function;
+    std::string source;
```

Comment:
Don't thing `source` belongs in this struct.

Reply:
We need something to carry the source of the `m_filename` we were given and pass it into plang::Script to be compiled. I'm not sure what you are suggesting with its removal.

Reply:
It's not an option that's handled in `addArgs`.  It can just be a variable on the stack at the point of assignment if I'm not mistaken.

Reply:
Ah. I kept the interface of `filters.numpy` because I suppose we might also want to directly pass in python source via arg rather than via m_filename. I was on the fence on that and left it off for now.